fetch.bs

```diff
@@ -7828,6 +7828,62 @@ resource — for non-<a>CORS requests</a> as well as <a>CORS
 requests</a> — and do not use `<code>Vary</code>`.
 
 
+<h2 id=fetch-elsewhere class=no-num>Fetch elsewhere</h2>
+
+<p>In its essence <a for=/>fetching</a> is an exchange of a <a for=/>request</a> for a
+<a for=/>response</a>. In reality it is rather complex mechanism for standards to adopt and use
+correctly. This section aims to give some advice.
+
+<p class=warning>Always ask domain experts for review.
+
+<p class=XXX>This is a work in progress.
+
+
+<h3 id=fetch-elsewhere-fetch class=no-num>Invoking fetch</h3>
+
+<p>Aside from a <a for=/>request</a> the <a for=/>fetch</a> operation takes several optional
+arguments. For those arguments that take an algorithm: the algorithm will be called from a task (or
+in a <a for=/>parallel queue</a> if <a for=fetch><i>useParallelQueue</i></a> is true).
+
+<dl>
+ <dt><a for=fetch><i>processRequestBody</i></a>
+ <dd><p>Takes an algorithm that will be passed the number of bytes that have been transmitted from
+ the <a for=/>request</a>'s <a for=request>body</a>. Most standards will not need this.
```

Comment:
Is this algo potentially called repeatedly during sending of the rquest, or just once?